plugins/lookup/laps.py

```diff
@@ -0,0 +1,233 @@
+# Copyright: (c) 2024, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+name: laps
+author: Nico Ohnezat (@no-12)
+short_description: Inventory plugin for Active Directory
+version_added: 2.2.0
+description:
+- Lookup plugin that retrieves the LAPS password information for multiple hosts from an Active Directory server.
+options:
+  _terms:
+    description:
+    - One or multiple C(CommonName) of the computer objects to search for.
+    required: True
+  laps:
+    description:
+    - The LAPS password type to retrieve.
+    - Defaults to the C(auto).
+    - This will attempt to retrieve the LAPS password in the following order C(windows_encrypted), C(windows_plain_text), C(legacy_microsoft).
```

Comment:
We should add a point saying the `dpapi-ng` Python library is required to decrypt the encrypted LAPS passwords.